peripherals/tx_wrapper/index.js

```diff
@@ -29,6 +29,11 @@ if (!process.env.EOS_SENDER) {
   process.exit();
 }
 
+if (!process.env.EOS_PERMISSION) {
+  console.log("Missing EOS_PERMISSION in .env file!");
```

Comment:
I was okay with it defaulting to `active` if not specified, but I can actually like it being explicit more.

Please update the README to capture the new `EOS_PERMISSION` and `EXPIRE_SEC` variables.

Reply:
updated [local_testnet_deployment_plan.md](https://github.com/eosnetworkfoundation/eos-evm/pull/503/commits/67d1ae8d8a424cb06a14864e27b9b56606f69ea3), please check again. thanks!

Reply:
Oh. I was referring to the README in peripherals/tx_wrapper.

Reply:
Modified README. Please check it again. thanks!